src/generate/device.rs

```diff
@@ -49,7 +54,7 @@ pub fn render(d: &Device, target: &Target, nightly: bool, device_x: &mut String)
         });
     }
 
-    match *target {
+    match target {
```

Comment:
Just a comment: this opts into the "match ergonomics" feature which may raise the MSRV -- not that it was ever documented or guaranteed.